vnext/ReactUWP/Views/DynamicAutomationPeer.cpp

```diff
@@ -345,6 +406,32 @@ bool DynamicAutomationPeer::GetAccessibilityState(winrt::PROJECT_ROOT_NAMESPACE:
   return false;
 }
 
+bool DynamicAutomationPeer::HasAccessibilityValue(winrt::PROJECT_ROOT_NAMESPACE::AccessibilityValue accValue) const {
+  try {
+    if (auto const &owner = Owner()) {
```

Comment:
>if (auto const &owner = Owner()) { [](start = 4, length = 34)

This null check isn't necessary...  The only ctor we have is the one that takes an owner, so you can assume it's never null.

Reply:
Removed for my changes.

Reply:
bonus points if you remove the exiting ones as well.

---
In reply to: [441096162](https://github.com/microsoft/react-native-windows/pull/5080#discussion_r441096162) [](ancestors = 441096162)